scripts/update/radarr.sh

```diff
@@ -44,8 +44,17 @@ if [[ -f /install/.radarr.lock ]]; then
             fi
 
         elif [[ $(_radarr_version) = "mono-v2" ]]; then
-            echo_warn "Please upgrade your radarr to v3. It will be migrated to .Net core on the next \`box update\` run"
+            echo_warn "Radarr v0.2 is EOL and not supported. Please upgrade your radarr to v3. It will be migrated to .Net core on the next \`box update\` run"
             echo_docs "applications/radarr#migrating-to-v3-on-net-core"
         fi
     fi
+    #Mandatory SSL Port change for Readarr
+    sslport=$(grep -oPm1 "(?<=<SslPort>)[^<]+" /home/"$radarrOwner"/.config/Radarr/config.xml)
+    sslenabled=$(grep -oPm1 "(?<=<EnableSsl>)[^<]+" /home/"$radarrOwner"/.config/Radarr/config.xml)
+    if $sslport '==' '8787'; then
+        sed s/\<SslPort\>port\=9898/\<SslPort\>port\=8787/ -i fil/home/"$radarrOwner"/.config/Radarr/config.xmle
+        if [ $sslenabled = true ]; then
+            echo "port changed for ssl due to possible future conflict with Readarr"
```

Comment:
```suggestion
            echo_info "Radarr SSL port changed from 9898 to 8787, please ensure to adjust your dependent systems in case they were using this port."
```